pkg/services/sqlstore/migrator/spanner_dialect.go

```diff
@@ -338,3 +342,41 @@ func SpannerConnectorConfigToClientOptions(connectorConfig spannerdriver.Connect
 func (s *SpannerDialect) UnionDistinct() string {
 	return "UNION DISTINCT"
 }
+
+func (s *SpannerDialect) findChangeStreams(ctx context.Context, engine *xorm.Engine) ([]string, error) {
+	dsn := engine.Dialect().DataSourceName()
+	cfg, err := spannerdriver.ExtractConnectorConfig(dsn)
+	if err != nil {
+		return nil, err
+	}
+
+	opts := SpannerConnectorConfigToClientOptions(cfg)
+	client, err := spanner.NewClient(ctx, dsn, opts...)
+	if err != nil {
+		return nil, err
+	}
+	defer client.Close()
```

Comment:
We're doing standard SQL query here, can't we use `engine.DB().Query("...")`, similar to queries in `dialect_spanner.go` for tables and indexes?

Reply:
It didn't even cross my mind. Updating